lib/govuk_request_id.rb

```diff
@@ -0,0 +1,13 @@
+class GovukRequestId
+  def self.insert(body)
+    return body unless body.present?
+
+    doc = Nokogiri::HTML::DocumentFragment.parse(body)
```

Comment:
Good call @gpeng 👍 
Not sure if this applies here, but there are servers that remove comments before serving the HTML so we might be having issues here?
I am not strongly attached to Nokogiri and the data attribute, but it feels to me like a bit more robust solution thinking in the long term?